lib/ronad/version.rb

```diff
@@ -0,0 +1,3 @@
+module Ronad
+  VERSION = "0.3.0"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)